docs/definition.md

```diff
@@ -180,6 +180,573 @@ There are a few terms you should be familiar with:
 7. `Success`, `Failure` and `Result` are markers returned by step
    methods to change business process execution path.
 
+### Invalid story definitions
+
+#### Story definitions which return a coroutine
+
+!!! danger "The story definition method itself must not be declared with `async def`"
+
+    Attempting to do so will raise a `StoryDefinitionError`.
+    
+    ```pycon tab="sync"
+
+    >>> from stories import Failure, Result, Success, arguments, story
+    
+    >>> class Subscription:
+    ...     """Buy subscription for certain category."""
+    ...
+    ...     @story
+    ...     @arguments("category_id", "price_id", "user_id")
+    ...     async def buy(I):
```

Comment:
Maybe this example is way too long to show the problem. Can we define it with fever stories steps?

Reply:
Sure, I was simply following along :)